drivers/include/adxl345.h

```diff
@@ -152,8 +154,6 @@ typedef struct {
  * @brief   Device descriptor for the ADXL345 sensor
  */
 typedef struct {
-    i2c_t i2c;                      /**< I2C device which is used */
-    uint8_t addr;                   /**< I2C address */
     adxl345_params_t *params;       /**< Device configuration */
```

Comment:
no need to use a pointer for params (was already the case)

Reply:
But it was requested by @haukepetersen in #6799 to save a bit of RAM.
The idea was the  keep the often used data in RAM and let the rest in ROM.

Reply:
I when through #6799 but couldn't find any reference of that. Looking at the device driver implementation (especially [this paragraph](https://github.com/RIOT-OS/RIOT/wiki/Guide:-Writing-a-device-driver-in-RIOT#device-descriptor-and-parameter-configuration)), the `params` attribute shouldn't be a pointer (it's called `p` in the wiki)

Reply:
I ACK if you change that @dylad ;)

Reply:
@aabadie see https://github.com/RIOT-OS/RIOT/pull/6799#discussion_r108861291
@haukepetersen what should we do ?

Reply:
@haukepetersen, what's your opinion here ? (You seem to wrote the wiki and give the pointer advice)

Reply:
I agree with using a pointer. This lets us put the configuration constants in ROM instead of in RAM. adxl345_params_t is significantly larger than a single pointer and I doubt the double dereference will cause any measurable performance delays when communicating with a I2C device.